warehouse/templates/pages/sponsors.html

```diff
@@ -142,7 +140,7 @@ <h2 class="centered-heading__title">Infrastructure sponsors</h2>
         {% if sponsor.infra_sponsor %}
         <div class="sponsor-grid__sponsor">
           <div class="sponsor-grid__sponsor-img">
-            <img src="{{ request.static_path('warehouse:static/dist/images/sponsors/color/' + sponsor.image) }}" alt="{{ sponsor.name }}">
+            <img src="{{ sponsor.color_logo_url }}" alt="{{ sponsor.name }}">
```

Comment:
[camoify](https://github.com/pypa/warehouse/blob/7fc3ce5bd7ecc93ef54c1652787fb5e7757fe6f2/warehouse/filters.py#L66-L85) this url